system/Images/Handlers/ImageMagickHandler.php

```diff
@@ -61,6 +65,21 @@ class ImageMagickHandler extends BaseHandler
 
 	//--------------------------------------------------------------------
 
+	public function __construct($config = null)
+	{
+		parent::__construct($config);
+
+		// We should never see this, so can't test it
+		// @codeCoverageIgnoreStart
+		if ( ! extension_loaded('imagick'))
```

Comment:
If it's not currently using the Imagick extension, we should remove this check. Good to have when it's refactored, though.